core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -69,17 +76,28 @@ oppia.controller('Dashboard', [
       $scope.checkForMobileView();
     });
 
+    $scope.sortExplorationsList = function(sortType) {
```

Comment:
Made `sortType` as an optional parameter, and also fixed the order being changed as ascending <-> descending whenever sortType is changed. But didn't use 2 functions, since they don't seem to be needed now. wdyt ?